paper-tabs.html

```diff
@@ -377,6 +378,39 @@
         }, 10);
       },
 
+      _onIronItemsChanged: function(event) {
```

Comment:
`iron-menu-behavior` already has [the listener for that event](https://github.com/PolymerElements/iron-menu-behavior/blob/0ba8b7d/iron-menu-behavior.html#L59), so here we don't really need to set it again. 
Here you're overriding a method that `iron-menu-behavior` will invoke, so it should be fine not to set it..!